spec/scoreCountersSpec.js

```diff
@@ -0,0 +1,18 @@
+// 'use strict';
+//
+// describe('ScoreCounters', function(){
+//   var scorecounters;
+//   var frame;
+//
+//   beforeEach(function(){
+//     frame = new Frame();
```

Comment:
I wonder if you've seen [this pill](https://github.com/makersacademy/course/blob/master/pills/mocking_in_javascript.md#use-a-fake-version-of-an-object-in-a-test)? Particularly the section "Use a fake version of an object in a test". Can you see how you could use that technique here to create a fake version of a Frame?